.github/workflows/test_s3_minio.yml

```
@@ -52,15 +58,24 @@ jobs:
           miniforge-version: "latest"
           miniforge-variant: Mambaforge
           use-mamba: true
-      - name: Install PyActiveStorage and run tests
-        shell: bash -l {0}
+      - name: Install PyActiveStorage
         run: |
           conda --version
           python -V
           which python
           pip install -e .
-          pytest
-          pytest tests/s3_exploratory/test_s3_reduction.py
+      - name: Run tests
+        run: |
+          pytest --numprocesses 1
```

This approach didn't fix the problem, so we could revert to parallel runs.
```suggestion
          pytest
```

---

indeed, we can use -n 2, lemme first fix the main issue - why the tests fail, then will cleanup :grin: